components/organisms/Repositories/repositories.tsx

```diff
@@ -95,105 +95,107 @@ const Repositories = ({ repositories }: RepositoriesProps): JSX.Element => {
         setRangeFilter={store.updateRange}
         title="Repositories"
       />
-      <div className="flex flex-col rounded-lg overflow-hidden border">
-        <div className="flex md:hidden justify-between  py-4 px-6 bg-light-slate-3 gap-2">
-          <div className="flex-1">
-            <TableTitle text="Repository"></TableTitle>
-          </div>
-          <div className="flex-1">
-            <TableTitle text="Pr Overview"></TableTitle>
-          </div>
-        </div>
-        <div className="hidden md:flex py-4 px-6 bg-light-slate-3 gap-2">
-          <div className={clsx(classNames.cols.checkbox)} >
-            <Checkbox label="" onChange={handleOnSelectAllChecked} disabled={!user} title={!user? "Connect to GitHub" : ""} className={`checked:[&>*]:!bg-orange-500 ${ user? "[&>*]:!border-orange-500 [&>*]:hover:!bg-orange-600": "[&>*]:!border-light-slate-8"}`}/>
-          </div>
-          <div className={clsx(classNames.cols.repository)}>
-            <TableTitle text="Repository"></TableTitle>
-          </div>
-          <div className={clsx(classNames.cols.activity)}>
-            <TableTitle text="Activity"></TableTitle>
-          </div>
-          <div className={clsx(classNames.cols.prOverview)}>
-            <TableTitle text="PR Overview"></TableTitle>
-          </div>
-          <div className={clsx(classNames.cols.prVelocity)}>
-            <TableTitle text="PR Velocity"></TableTitle>
-          </div>
-          <div className={clsx(classNames.cols.spam)}>
-            <TableTitle text="SPAM"></TableTitle>
-          </div>
-          <div className={clsx(classNames.cols.contributors, "hidden lg:flex")}>
-            <TableTitle text="Contributors"></TableTitle>
+      <div className="flex flex-col rounded-lg overflow-x-auto border w-full">
+        <div className="lg:min-w-[1150px]">
```

Comment:
this is the minimum value for a fully responsive table and a breakpoint was used to prevent the min-width from creating a horizontal scroll on medium and small screen table layouts

Reply:
agree with the class, not sure wrapping all the content in a wrapper is needed tho since we want to override `w-full` with `lg:min-w-[1150px]`

Reply:
with the implementation, a div was used to prevent the table header from scrolling with the table when a horizontal scroll is happening on the table, and I think that is a good approach on this one 👍